python/paddle/v2/dataset/common.py

```diff
@@ -27,7 +28,12 @@
 DATA_HOME = os.path.expanduser('~/.cache/paddle/dataset')
 
 if not os.path.exists(DATA_HOME):
-    os.makedirs(DATA_HOME)
+    try:
+        os.makedirs(DATA_HOME)
+    except OSError as exc:
+        if exc.errno != errno.EEXIST:
```

Comment:
Does this mean that we can remove the check for existence in L30 and just run the try block?